node/core/backing/src/lib.rs

```diff
@@ -1169,6 +1169,7 @@ impl util::JobTrait for CandidateBackingJob {
 	type Metrics = Metrics;
 
 	const NAME: &'static str = "CandidateBackingJob";
+	const SUBSYSTEM: &'static str = "candidate_backing";
```

Comment:
Can't we re-use the `NAME`?

Reply:
The subsystem names are generated from the `Overseer` field names in `impl_builder`. I'll get back to this once we settle on how we name them 😁

Reply:
Switched the name to kebab case. We can now definitely reuse the name here: `candidate-backing` for both.

Reply:
implemented.